src/core/msg.rs

```diff
@@ -7,7 +7,7 @@ use serde::{Deserialize, Serialize};
 use super::types::access_route::AccessRoute;
 
 /// The struct used to instantiate the contract.  Utilized in the core [contract file](crate::contract::instantiate).
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
```

Comment:
These `Eq` additions conform to clippy 1.63.0 https://rust-lang.github.io/rust-clippy/master/index.html#derive_partial_eq_without_eq